esp/services/ws_sql/ws_sqlService.cpp

```diff
@@ -2003,7 +2003,11 @@ bool CwssqlEx::isValidCluster(const char *cluster)
     CriticalBlock block(crit);
     if (validClusters.getValue(cluster))
         return true;
+#ifndef _CONTAINERIZED
     if (validateTargetClusterName(cluster))
+#else
+    if (validateContainerTargetClusterName(cluster))
+#endif
```

Comment:
minor: good to minimize the amount of conditional code if possible/clean to do so.
As this is used here and in ws_workunitsService below (and prob. other places in future?), it would be cleaner if a common method (e.g. add a 'validateTargetName' to TpWrapper) contained this conditional code, so that each use like this didn't need to have conditional code in it.

Reply:
I guess I can move the whole isValidCluster() (and the conditional code) into a new validateTargetName().

Reply:
The code has been modified to share the code from both ws_workunitsService and ws_sql.